vllm/config.py

```diff
@@ -299,6 +300,30 @@ def _verify_quantization(self) -> None:
                     f"method specified in the `quantization` argument "
                     f"({self.quantization}).")
 
+        # TODO(alpin): Tune the default exp_bits for each fp_bits
+        DEFAULT_EXP_BITS = {
+            4: 2,
+            5: 2,
+            6: 2,
+            7: 3,
+        }
+        quant_llm_methods = [
+            "fp4_weights", "fp5_weights", "fp6_weights", "fp7_weights"
+        ]
+        if self.quantization is not None and self.quantization in \
+            quant_llm_methods:
+            fp_bits = int(self.quantization[2])
+            exp_bits = DEFAULT_EXP_BITS[fp_bits]
+            self.hf_config.quantization_config = {
+                "bits": fp_bits,
+                "exp_bits": exp_bits,
+                "quant_method": self.quantization
+            }
+            # TODO(alpin): Investigate supporting bfloat16 dtype
+            self.dtype = torch.float16
+            # In some cases, CUDA graph execution breaks this quant method
+            self.enforce_eager = True
```

Comment:
Both of these are worth warnings, for instance bitsandbytes has one for enforce_eager since it cannot support cuda graphs on older versions. It would be worth investigating to enable cuda graphs across the board since it is important for performance